apps/testrunner/queue.py

```diff
@@ -0,0 +1,105 @@
+import json
+import time
+
+import requests
+import sys
+
+from .logger import logger
+from .runner import SimpleTest
+
+
+class TaskRepo(object):
+    # todo: add HTTP requests to fetch/save data
+    URL = 'http://localhost:8000/task'
+
+    def acquire(self):
+        url = '{}/acquire'.format(self.URL)
+        response = requests.get(url)
+        if response.ok:
+            task_data = response.json()
+            return Task(self, task_data['id'], task_data)
+
+    def release(self, task):
+        url = '{}/{}/lock'.format(self.URL, task.id)
```

Comment:
I thought we planned to use django magic to find out exact URLs and the current implementation is just a result of me not knowing how to fetch this data from django.